nixos/modules/services/networking/netbird.nix

```diff
@@ -1,171 +1,383 @@
+              };
+
+              firewall.udp.ports = mkOption {
+                type = listOf port;
+                default = [ ];
+                description = ''List of open UDP ports.'';
+              };
+
+              firewall.udp.ranges = mkOption {
+                type = listOf (attrsOf port);
+                default = [ ];
+                example = [{ from = 8999; to = 9003; }];
+                description = ''Range of open UDP ports.'';
+              };
+
+              wrappers = mkOption {
+                type = attrsOf package;
+                internal = true;
+                default =
+                  let
+                    exports = pipe config.environment [
+                      (mapAttrsToList toShellVar)
+                      (map (assignment: "export ${assignment}"))
+                      (concatStringsSep "\n")
+                    ];
+                    mkWrapper = tool: pkgs.writeShellScriptBin "${tool}${config.suffix}" ''
+                      ${exports}
+                      ${lib.getExe' cfg.package tool} "$@"
+                    '';
+                    tools = [ "netbird" "netbird-mgmt" "netbird-signal" ];
+                  in
+                  {
+                    netbird = mkWrapper "netbird";
+                    netbird-mgmt = mkWrapper "netbird-mgmt";
+                    netbird-signal = mkWrapper "netbird-signal";
+                  };
               };
             };
 
-            config.environment = builtins.mapAttrs (_: mkDefault) {
-              NB_CONFIG = "/var/lib/${config.stateDir}/config.json";
-              NB_LOG_FILE = "console";
-              NB_WIREGUARD_PORT = builtins.toString config.port;
-              NB_INTERFACE_NAME = name;
-              NB_DAEMON_ADDR = "unix:///var/run/${config.stateDir}/sock";
+            config.environment = {
+              NB_CONFIG = "/var/lib/${config.name}/config.json";
+              NB_DAEMON_ADDR = "unix:///var/run/${config.name}/sock";
+              NB_INTERFACE_NAME = config.interface;
+              NB_LOG_FILE = mkOptionDefault "console";
+              NB_LOG_LEVEL = config.logLevel;
+              NB_WIREGUARD_PORT = toString config.port;
             };
           }
         )
       );
       default = { };
       description = ''
-        Attribute set of Netbird tunnels, each one will spawn a daemon listening on ...
+        Attribute set of Netbird clients, each one will spawn an daemon listening on a chosen UDP port.
+
+        A single daemon is fully operational.
+
+        Multiple daemons will interfere with each other's DNS resolution, but otherwise should remain operational.
+      '';
+      example = lib.literalExpression ''
+        {
+          services.netbird.clients.default.port = 51820;
+          services.netbird.clients.personal.port = 51821;
+          services.netbird.clients.work1.port = 51822;
+        }
       '';
     };
   };
 
   config = mkMerge [
-    (mkIf cfg.enable {
```

Comment:
I think we should keep this. Once multi-account connections are implemented in netbird itself, the update should be a no-op for the majority of users. There's also a section for NixOS in their documentation, we should keep supporting it: https://docs.netbird.io/how-to/getting-started#linux

Reply:
nice find on that `meta.doc`, will also help me documenting the other PR I have opened :)